src/functionaltests/java/com/ericsson/ei/subscriptions/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -0,0 +1,229 @@
+        JsonNode responseEntityNode = jsonNodeResult.get("responseEntity");
+        String responseEntityNodeFormatted = responseEntityNode.asText().replace("[", "").replace("]", "");
+        JsonNode responseEntityFormattedJsonNode = new ObjectMapper().readValue(responseEntityNodeFormatted, JsonNode.class);
+        
+        LOGGER.debug("AggregatedObject from Response: " + responseEntityFormattedJsonNode.toString());
+       
+        String actualTestCaseStartedEventId = responseEntityFormattedJsonNode.get("testCaseExecutions").get("testCaseStartedEventId").asText();
+        assertEquals(HttpStatus.OK.toString(), Integer.toString(mvcResult.getResponse().getStatus()));
+        assertEquals("Diffences between actual Missed Notification response:\n" + expectedTestCaseStartedEventId
+                + "\nand expected  Missed Notification response:\n" + actualTestCaseStartedEventId,
+                expectedTestCaseStartedEventId, actualTestCaseStartedEventId);
+        
+    }
+
+    @And("^Check missed notification has been returned$")
+    public void check_missed_notification_has_been_returned() throws Throwable {
+        String expectedResponse = "{\"responseEntity\":\"[]\"}";
+        String subscriptionName = "Subscription_1";
+        String entryPoint = "/queryMissedNotifications";
+        
+        LOGGER.debug("Trying to query /queryMissedNotifications RestApi one more time with subscriptionName: " + subscriptionName);
+        mvcResult = mockMvc.perform(get(entryPoint)
+                .param("SubscriptionName", subscriptionName)
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andReturn();
+        String responseAsString = mvcResult.getResponse().getContentAsString();
+        int reponseStatusCode = mvcResult.getResponse().getStatus();
+        LOGGER.debug("Response of /queryMissedNotifications RestApi, Status Code: " + reponseStatusCode +
+                "\nResponse: " + responseAsString);
+        
+        assertEquals(HttpStatus.OK.toString(), Integer.toString(reponseStatusCode));
+        assertEquals("Diffences between actual Missed Notification response:\n" + responseAsString
+                + "\nand expected  Missed Notification response:\n" + expectedResponse,
+                expectedResponse, responseAsString);
+    }
+    
+    private boolean createDocumentInMongoDb(String databaseName, String collectionName, String objToBeInserted) {
+        LOGGER.debug("Inserting Object to MongoDb.\nDatabase: " + databaseName
+                     + "\nCollection: " + collectionName
+                     + "\nDocument to be inserted\n: " + objToBeInserted);
+        return mongoDBHandler.insertDocument(databaseName, collectionName, objToBeInserted);
+    }
+    
+    private String getDocumentIdBasedOnQuery(String dataBaseName, String collectionName, String query) {
```

Comment:
I'm thinking this function can be put in the FunctionalTestBase class, as 3 different functional tests (perhaps more in the future?) are using this.